zuul.d/adoption.yaml

```diff
@@ -125,9 +131,15 @@
       - OWNERS
       - .*/*.md
 
+# TODO(marios): remove this once we finish rename OSPRH-8452
 - job:
     name: cifmw-data-plane-adoption-osp-17-to-extracted-crc-minimal-no-ceph
-    parent: cifmw-data-plane-adoption-osp-17-to-extracted-crc
+    parent: adoption-standalone-to-crc-no-ceph-provider
+
+- job:
+    name: adoption-standalone-to-crc-no-ceph-provider
+    description: Standalone source OSP 17.1 node adopted to crc RHOSO 18. Has TLS, not ceph. Uses a content provider.
```

Comment:
same comment regarding description.

Reply:
ack

just wanted it to stay one line to be honest and it was short enough i felt, but ok updating